jujugui/static/gui/src/app/components/entity-content/entity-content.js

```diff
@@ -230,14 +240,20 @@ YUI.add('entity-content', function() {
                   renderMarkdown={this.props.renderMarkdown}
                   getFile={this.props.getFile} />
               </div>
-              {this._showEntityRelations()}
               <div className="four-col">
-                <juju.components.EntityFiles
-                  apiUrl={this.props.apiUrl}
-                  entityModel={entityModel}
-                  pluralize={this.props.pluralize} />
+                {this._showEntityRelations()}
+                <div className="four-col">
```

Comment:
We don't need this `div.four-col` as its wrapped in a four-col already and resulting in extra gap between blocks.